src/components/molecules/call-card.tsx

```diff
@@ -26,6 +26,7 @@ const styles = StyleSheet.create({
     alignItems: 'center'
   },
   text: {
-    paddingLeft: 20
+    paddingLeft: 20,
+    maxWidth: 260
```

Comment:
How did we get to this value? Are we limiting the card itself regardless of the screen size?

Reply:
I adjusted it from between 300 & 200 to find the lowest number that works on all languages.
It doesn't affect the card as a whole just the text inside the card

Reply:
I worry that this is not a scalable solution as we add more languages and run on more screen sizes. Rather than give a specific width, can we find a way to allow it to take up the full width available and use margins/padding on a parent to confine the card overall?

Reply:
Agreed - I would wrap the Text element in a View and set that View to be `flex: 1` to be the dominant element in the card.

Reply:
Sounds like a plan I'll make that change now